index.js

```diff
@@ -58,6 +59,7 @@ module.exports = {
     'react',
     'react-hooks',
     'you-dont-need-lodash-underscore',
+    'inclusive-language',
```

Comment:
I _think_ this list is alphabetical?

Reply:
Fixed in c8c7ddafed3adcccf5e364e33139c933b894b6f9.